### PR TITLE
[enhance](Hdfs) Add bvar for currently inflight hdfs file writer's num

### PR DESCRIPTION
**Pull request by ByteYue**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by ByteYue** [comment]
run buildall

**Review by gavinchou — APPROVED** [review]

**Review by platoneko — APPROVED** [review]

**Review by platoneko — CHANGES_REQUESTED** [review]
What's the difference between `hdfs_file_being_written` and `inflight_hdfs_file_writer`?

**Comment by ByteYue** [comment]
run buildall

File: be/src/io/fs/hdfs_file_writer.cpp [hunk 1]
```diff
@@ -150,14 +149,14 @@ HdfsFileWriter::~HdfsFileWriter() {
         // For thread safety
         std::ignore = _async_close_pack->promise.get_future();
         _async_close_pack = nullptr;
+        inflight_hdfs_file_writer << -1;
```
Comment (platoneko) [inline be/src/io/fs/hdfs_file_writer.cpp:152]: Duplicate?

**Comment by ByteYue** [comment]
run buildall

**Review by platoneko — APPROVED** [review]

**Pull request merged**